### PR TITLE
Update QA scenarios to use mariadb as default database.

### PR DESCRIPTION
**Pull request by gosipyan**
Keep postgresql for two nodes cluster setup

### CODE REVIEW

**Comment by jsuchome** [comment]
I do not think we need postgres jobs for Cloud8...

**Comment by gosipyan** [comment]
@jsuchome I am not sure what do to with scenarios which had 3 cluster each 2 nodes because maximum nodes for each QA setup is 7 .But I will update others  which have one cluster with two nodes.

**Comment by jsuchome** [comment]
Hm, true. Well, it looks like we need to reevaluate what needs to be tested. Is there some list somewhere of what is required to be covered by these tests?

**Comment by gosipyan** [comment]
@jsuchome so far as I can see that all have drbd enabled so if we wont have it in cloud8 anymore maybe I can update them and remove it.

**Comment by gosipyan** [comment]
@vmoravec Regarding nfsvers I decided do some cleanup also ,because I remember we are talking about this already since cloud7 that no need use this option @kbaikov maybe can give more info.Can revert if needed.
I am planing run each of them before merge to make sure nothing broken and update pr with results.

**Comment by gosipyan** [comment]
https://ci.suse.de/view/Cloud/view/QA/job/cloud-mkphyscloud-qa-scenario-1a/666/
https://ci.suse.de/view/Cloud/view/QA/job/cloud-mkphyscloud-qa-scenario-1b/345/

**Comment by gosipyan** [comment]
@vmoravec Can you have look again. I reduce number of changed scenarios  .And all changes were deployed successfully .You can see links in above comment.

File: scripts/scenarios/cloud8/qa/no-ssl/qa-scenario-3.yaml [hunk 1]
```diff
@@ -10,7 +10,6 @@ proposals:
         export: ##shared_nfs_export_for_glance##
         mount_path: "/images"
         mount_options:
-        - nfsvers=3
```
Comment (vmoravec) [inline scripts/scenarios/cloud8/qa/no-ssl/qa-scenario-3.yaml]: Can we remove also the attribute key `mount_options`, having it there empty will be confusing. The same is the case in 6a scenario.

Comment (gosipyan) [reply scripts/scenarios/cloud8/qa/no-ssl/qa-scenario-3.yaml]: Fixed

**Pull request merged**